presto-cli/src/main/java/com/facebook/presto/cli/ErrorMessages.java

```diff
@@ -66,13 +66,6 @@ private static String runtimeExceptionErrorMessage(Throwable throwable, ClientSe
         return builder.toString();
     }
 
-    private static String getStackTraceString(Throwable throwable)
```

Comment:
Unrelated change? Why did this move? (It make it harder to review, since it's not clear whether there are any actual changes)